### PR TITLE
PLT-2798 Config sanitation fails when no LDAP Settings are present in…

### PR DESCRIPTION
**Pull request by enahum**
… config.json

### CODE REVIEW

**Comment by coreyhulen** [comment]
@jwilander is this needed for the other FAKE_SETTING ?

File: model/config.go [hunk 1]
```diff
@@ -621,7 +621,7 @@ func (o *Config) GetSanitizeOptions() map[string]bool {
 }
 
 func (o *Config) Sanitize() {
-	if len(*o.LdapSettings.BindPassword) > 0 {
+	if &o.LdapSettings == nil && len(*o.LdapSettings.BindPassword) > 0 {
```
Comment (crspeller) [inline model/config.go]: Should this not be `!= nil`? Or am I misunderstanding something.

Comment (enahum) [inline model/config.go]: absolutely.. fixing it

**Comment by jwilander** [comment]
Nope, this is the only one that was a pointer

**Comment by jwilander** [comment]
+1

**Pull request merged**